Makefile

```diff
@@ -93,22 +92,16 @@ endif
 
 
 .PHONY: install.tools
-install.tools: .install.golint .install.govet .install.gitvalidation
+install.tools: .install.golint .install.gitvalidation
 
 # golint does not even build for <go1.5
 .install.golint:
 ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get github.com/golang/lint/golint
-endif
-
-# go vet is now included in >=go1.5, so no need to get it.
-.install.govet:
-ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get golang.org/x/tools/cmd/vet
```

Comment:
If you drop this, you'll also want to rewrite/remove the line in the `.govet` recipe with the:

> ERROR: 'go vet' not found. Consider 'make install.tools' target

message.